AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -306,6 +306,13 @@ extern NSInteger const ASDefaultDrawingPriority;
  */
 @property (nonatomic, readonly, assign) ASSizeRange constrainedSizeForCalculatedLayout;
 
+/**
+ * @abstract Sublcass hook for nodes that are acting as root nodes. This method get's called if one of the subnodes
+ * size is invalidated and may need to result in a different size as the current calculated size.
+ */
+- (void)displayNodeDidInvalidateSizeNewSize:(CGSize)newSize;
```

Comment:
Moved to `ASDisplayNode+FrameWorkPrivate.h`. Just gonna be interesting if we would allow to write custom containers, but in that case we can than improve the API a bit more and ship some documentation with it.